perception/euclidean_cluster/src/voxel_grid_based_euclidean_cluster_node.cpp

```diff
@@ -26,22 +26,21 @@ VoxelGridBasedEuclideanClusterNode::VoxelGridBasedEuclideanClusterNode(
 {
   const bool use_height = this->declare_parameter("use_height", false);
   const int min_cluster_size = this->declare_parameter("min_cluster_size", 1);
-  const int max_cluster_size = this->declare_parameter("max_cluster_size", 500);
+  const int max_cluster_size = this->declare_parameter("max_cluster_size", 1024);
   const float tolerance = this->declare_parameter("tolerance", 1.0);
-  const float voxel_leaf_size = this->declare_parameter("voxel_leaf_size", 0.5);
+  const float voxel_leaf_size = this->declare_parameter("voxel_leaf_size", 0.01);
   const int min_points_number_per_voxel = this->declare_parameter("min_points_number_per_voxel", 3);
   cluster_ = std::make_shared<VoxelGridBasedEuclideanCluster>(
     use_height, min_cluster_size, max_cluster_size, tolerance, voxel_leaf_size,
     min_points_number_per_voxel);
 
+  detected_objects_pub_ = this->create_publisher<autoware_auto_perception_msgs::msg::DetectedObjects>(
+    "detected_objects_universe", rclcpp::QoS{1});
```

Comment:
Please don't use `universe` in names. Our stack is all one. No need to differentiate between Universe and Auto.

Reply:
In fact, the original `output` is just fine because it will almost always be remapped.

Reply:
the name was changed to default "output" after https://github.com/upowerrobotics/autoware.universe/pull/4/commits/9d6afe6fb52c8cfe54957f9077d8eba927b6b4c1